spec/system/cohorts/student_report_spec.rb

```diff
@@ -223,6 +226,16 @@
     expect(page).to have_text("Cohort")
     expect(page).to have_text(cohort.name)
 
+    expect(page).to have_text("Standing")
```

Comment:
Unrelated to this file, but this reminded me that it would be good to have the _current standing_ added to the course export's student details sheet.

Reply:
Updated https://github.com/pupilfirst/pupilfirst/pull/1468/commits/717b3e587f07d917d3a8e373a4f9f3aa192f85d0